q2_chemistree/_fingerprint.py

```diff
@@ -130,7 +131,8 @@ def fingerprint(sirius_path: str, features: MGFDirFmt, ppm_max: int, profile: st
     cmdsir = [str(sirius), '--quiet',
               '--initial-compound-buffer', str(1),
               '--max-compound-buffer', str(32), '--profile', str(profile),
-              '--database', str(database), '--candidates',  str(num_candidates),
+              '--database', str(database),
+              '--candidates',  str(num_candidates),
```

Comment:
can you remove the extra space before `str`. Looks like there's two spaces should only be one.

Reply:
updated PR! :)